WebDriverAgentLib/Utilities/FBConfiguration.m

```diff
@@ -44,6 +44,7 @@
 static BOOL FBIncludeNonModalElements = NO;
 static NSString *FBAcceptAlertButtonSelector = @"";
 static NSString *FBDismissAlertButtonSelector = @"";
+static BOOL FBShouldUseLargeAlertReverseAction = YES;
```

Comment:
I assume it should be disabled by default in order to keep the legacy behavior

Reply:
I thought we were going to change the default behavior, do you think we should we retain the previous one?